gbus/abstractions.go

```diff
@@ -142,6 +146,13 @@ type SagaRegister interface {
 	RegisterSaga(saga Saga, conf ...SagaConfFn) error
 }
 
+//SagaGlue glues together all the parts needed in order to orchistrate saga instances
+type SagaGlue interface {
```

Comment:
You add `Logged` as part of the interface.

Reply:
done